### DISCUSSION TITLE
Scan Metrics not working

### DISCUSSION THREAD

**Discussion by SecTheBit** [projectdiscovery/nuclei#2147] [Q&A] [answered]
Hi Team,

I ran nuclei with the metrics flag and then checked the stats on default port 9092, but I didn't get any output from there. (Refer below screenshot)

![Screenshot from 2022-06-10 17-12-04](https://user-images.githubusercontent.com/46895441/173059349-52ee4505-ecc7-4681-be16-a6ae6cb1dac2.png)

**Comment by ehsandeep** [comment, marked as answer]
@SecTheBit this endpoint is accessible during the scan time not after scan gets completed.

**Reply by SecTheBit** [reply]
@ehsandeep  Is that possible that we can get this metrics even after the scanning, or save the metrics output in json format?

**Reply by ehsandeep** [reply]
@SecTheBit Following up https://github.com/projectdiscovery/nuclei/issues/2160, what's the use case of writing the stats data to file?
